test/es-module/test-esm-import-meta.mjs

```diff
@@ -3,7 +3,7 @@ import assert from 'assert';
 
 assert.strictEqual(Object.getPrototypeOf(import.meta), null);
 
-const keys = ['url'];
+const keys = ['__dirname', '__filename', 'url'];
```

Comment:
should have negative test for `data:` at least since it is always supported.

Reply:
I don't follow. What is `data:` referring to?

Reply:
https://nodejs.org/dist/latest-v20.x/docs/api/esm.html#data-imports

Reply:
I have added a test in https://github.com/nodejs/node/pull/48740/commits/dac12b1e43816603829a0a7c4392745a3e55214f that I believe covers this concern.